### DISCUSSION TITLE
Améliorer la sécurité des Startups d'État

### DISCUSSION THREAD

**Discussion by sneko** [betagouv/cap#25] [Besoin]
### Besoin

```bash
Chaque SE travaille en autonomie et il est difficile d'évaluer la sécurité des services.
```

### Solutions envisageables

```bash
1. BugBounty
2. Outils automatisés (pentesting)
```

### Informations complémentaires

- [X] J'avais déjà entendu ce besoin au sein de la communauté beta.gouv
- [X] Je me porte volontaire pour participer à la mise en place de la future solution choisie

**Comment by revolunet** [comment]
related: https://github.com/betagouv/cap/discussions/31

**Comment by revolunet** [comment]
Bonne nouvelle, nous avons enfin un recrutement en cours sur ce sujet
